Shell/Twee.cpp

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
If this is now the "Twee goal transformation" should this file be `TweeGoalTransformation.cpp`?

Reply:
Done!